ansys/dpf/core/custom_type_field.py

```diff
@@ -38,7 +38,7 @@ def __missing__(self, key):
 
 class CustomTypeField(_FieldBase):
     """Represents a simulation data container with each unitary data being of a custom type.
-    When initiliazing the CustomTypeField, a unitary data type should be given.
+    When initializing the ``CustomTypeField``, provide a unitary data type.
```

Comment:
```suggestion
    When initializing the ``CustomTypeField`` class, provide a unitary data type.
```
literal strings are supposed to have a noun after them to indicate their types. So--there was a missing word.

Reply:
@PipKat sorry about that. The reason why the DPF libraries have these errors and not other repositories is because some pre-commit and doc style verification tools are not yet set-up (such as one checking the Google developer style is followed). These errors are automatically flagged for other PyAnsys repositories. This is something which will be implemented next week for DPF repositories.

Reply:
This is revolved, however it looks like GitHub does not want to keep the "resolved" status.